src/mol-plugin/layout.ts

```diff
@@ -23,6 +23,7 @@ export type PluginLayoutControlsDisplay = 'outside' | 'portrait' | 'landscape' |
 export const PluginLayoutStateParams = {
     isExpanded: PD.Boolean(false),
     showControls: PD.Boolean(true),
+    showSecondaryStructureSequence: PD.Boolean(false),
```

Comment:
Please move this to `mol-plugin-ui`'s  `PluginUISpec.components.sequenceViewer` as a `view?: React.ComponentClass`.